recipes/default.rb

```diff
@@ -194,3 +194,84 @@
     source "aws-smtp-relay.erb"
     mode "0755"
 end
+
+# Installing Supervisor via yum gives initd integration, but has import problems.
+# Installing via pip fixes the import problems, but doesn't provide the integration.
+# So we do both.
+execute "pip --cache-dir=/tmp/ install supervisor"
```

Comment:
could that only be an issue with the os we are currently using and a more current gen os won't have any issues?

Reply:
Actually, moving to Supervisor the initd integration should be redundant. I'll revise.